front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -622,7 +622,7 @@ export function AgentMessage({
           />
 
           {agentMessage.chainOfThought?.length ? (
-            <ContentMessage title="Agent thoughts" variant="slate">
+            <ContentMessage title="Agent thoughts" variant="info">
```

Comment:
@Duncid updated to `primary` in this PR 😄 https://github.com/dust-tt/dust/pull/11806/files

Reply:
Should be yeah